### PR TITLE
allow multiple ga keys

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -124,7 +124,7 @@
 
 USE_TZ = True
 
-GA_TAG_KEY = env.str("GOOGLE_GA_TAG_KEY", "REPLACE_ME")
+GA_TAG_KEYS = env.str("GOOGLE_GA_TAG_KEYS", "REPLACE_ME")
```
Comment (rudigiesler) [inline config/settings/base.py]: There's an `env.list` which will handle splitting a `,` separated value into a list for us.

Comment (rudigiesler) [reply config/settings/base.py]: We can probably also change the default to an empty list `[]`, then we can get rid of the `if` in the template

File: higher_health/tests/test_context_processors.py [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
-from higher_health.context_processors import api_keys
+from higher_health.context_processors import api_keys, ga_tags
 
 
 class ApiKeysTestCase(TestCase):
     def test_api_keys(self):
         self.assertEqual(api_keys(None), {"PLACES_API_KEY": "TEST_API_KEY"})
+
+    def test_multiple_api_keys(self):
+        self.assertEqual(api_keys(None), {"PLACES_API_KEY": "TEST_API_KEY"})
```
Comment (rudigiesler) [inline higher_health/tests/test_context_processors.py]: This seems like a duplicate of the `test_api_keys` test.

File: config/settings/test.py [hunk 3]
```diff
@@ -13,7 +13,7 @@
 ENV_HOSTS = [host for host in env.str("ALLOWED_HOSTS", "").split(",") if host]
 ALLOWED_HOSTS = ENV_HOSTS + ["localhost", ".localhost", "127.0.0.1", "0.0.0.0"]
 
-GA_TAG_KEY = "TEST_GA_TAG_KEY"
+# GA_TAG_KEYS = env.list("GOOGLE_GA_TAG_KEYS", default=["TEST_GA_TAG_KEY"])
```
Comment (rudigiesler) [inline config/settings/test.py]: Is there a reason that this is commented out?

**Review by rudigiesler — APPROVED** [review]

File: config/settings/test.py (hunk 3 above)
Comment (miltontony) [reply config/settings/test.py]: Yes. We have it defined in the `base.py` so it's not needed here anymore. I've updated the tests to use the default value . I'll remove the comment

**Pull request merged**